packages/core/sonar-project.properties

```diff
@@ -0,0 +1,16 @@
+sonar.projectKey=inrupt_solid-client-authn-core
+sonar.organization=inrupt
```

Comment:
Do we need projectName here too?

Reply:
Not necessary but I will add it for consistency in this project. It was only required in one place to fix the incorrect name that was used when the project was first imported.